### ISSUE TITLE
Jarvis Stating Successful but not playing music in Music Assitant

### ISSUE THREAD

**Issue by adamtar** [FutureProofHomes/wyoming-enhancements#46]
So I check every setting and I am not sure what's wrong. I don't see any error codes. But, When I command jarvis to play music nothing changes in music assistant. But, it confirms it was played correctly. Has anyone else had this issue before? TIA

**Comment by debdalerichard** [comment]
can you get it play through your wyoming satellite any time? or just while you verbally instruct it to play?? also have you tried inputting text in Assist in your web browser?

**Comment by adamtar** [comment]
@debdalerichard Yes, it plays through the satellite manual at any time. Verbally instructed it says it is playing at the satellite location but it does nothing in music assistant. I am using gpt-3.5-turbo-1106. Thank you.

**Comment by debdalerichard** [comment]
I've sometimes go to instruct command twice?? have you tried typing the request in with the assist GUI (top R/H corner) and see how chatGPT-3.5-turbo response?

**Comment by adamtar** [comment]
Yes. It confirms it successfully playing. I go to check music assistant and nothing is searched or in the queue. It’s almost like music assistant isn’t getting the search requests. I am using Apple Music and YouTube music.

**Comment by debdalerichard** [comment]
I've just used Local NAS drive ..I've never used online music
sources?..sorry I can't help you any further..

**Comment by mkcologne** [comment]
Same Problem.
Everything worked finde till I've setup another satellite.

Getting an error:

```Error executing script. Invalid data for call_service at pos 2: Script requires 'response_variable' for response data for service call mass.search ```

When I ask assist what variables he used for the query it outputs the right ones 
{{music_query}} and {{mass_media_player}}

**Comment by adamtar** [comment]
@debdalerichard maybe you can help me with this. I just discovered that as long as I have music currently playing on a device I can request using Jarvis, and it works. But if nothing is playing or in the queue, it won't work. Any thoughts on how to fix this ?TIA

**Comment by mkcologne** [comment]
My solution:

I figured out, that the conversation process is build in in MA now.
So I deleted the script and the "spec" lines in my Assist settings and everything is working fine.
